packages/theme/src/abstracts/_variables-bootstrap.scss

```diff
@@ -84,14 +86,17 @@ $font-weight-bold: 600;
 
 //** Unit-less `line-height` for use in components like buttons.
 $line-height-base: 1.5; // 24/16
+$line-height-small: 1.7; // 24/14
```

Comment:
Where will this new line-height be used? I do find the naming of it to be confusing since it's _larger_ than the `-base` variant.

Reply:
Our input labels are 14px font, but still have a 24px line-height. The line-height is larger to account for the smaller font size. The small refers to "body small" text elements. Perhaps "$line-height-body-small" makes more sense?

Reply:
This makes sense, perhaps we just add a comment clarifying that it is body-small, but keep the naming convention as you have it.

Additionally, we should remove the second definition which is on line `149`.

Reply:
It looks like '$line-height-small'  on line '149' is used in some other mixins to calculate large and small form-groups (ones we're not currently using because we only have a single form size). Maybe it would be best to leave those defaults intact and I can update my new variable name to '$line-height-body-small'?